tests/scripts/packages/lib/isGardener.spec.ts

```diff
@@ -3,7 +3,7 @@ import isGardener from 'lib/isGardener';
 
 test.describe('isGardener', () => {
   test('should return true if the ID is included in the gardener list', () => {
-    expect(isGardener('0x0d')).toBeTruthy();
+    expect(isGardener('0x03')).not.toBeTruthy();
```

Comment:
It does not match the test description above, I don't understand this test then 🤷‍♂️

Reply:
i probably should skip it

Reply:
You need to edit this file https://github.com/ConsenSys/lineaster/blob/main/packages/data/gardeners.ts and add `0x03` to the list (i think you need to edit the `testnetGardeners` not the `mainnet` one as it was initially)